src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -74,11 +78,46 @@
   static final String ENCRYPTED_FILE_EXTENSION = ".enc";
   static final String MISSING_FILE_EXTENSION = ".missing";
 
+  static final File cacheFile = new File(getFormsFolder().getParent(), "cache.ser");
```

Comment:
The .ser filetype is reserved for the output of serialization from Java apps. I personally like "cache.ser" but I don't mind if it's renamed.